documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -121,27 +121,21 @@ The default value is "PLAINTEXT".
 You can set the value to `PLAINTEXT`, `SSL`, `SASL_PLAINTEXT`, or `SASL_SSL`.
 
 . If you want to connect to Kafka brokers that are using certificates from a public certificate authority, set `STRIMZI_PUBLIC_CA` to `true`. Set this property to `true`, for example, if you are using Amazon AWS MSK service.
-. If you enabled TLS with the `STRIMZI_TLS_ENABLED` environment variable, specify the keystore and truststore used to authenticate connection to the Kafka cluster.
+. If you enabled TLS with the `STRIMZI_TLS_ENABLED` environment variable, specify names of the secrets containing public and private keys used to authenticate connection to the Kafka cluster.
 +
 .Example TLS configuration
 [source,shell,subs=+quotes]
 ----
 # ....
 env:
-  - name: STRIMZI_TRUSTSTORE_LOCATION <1>
-    value: "/path/to/truststore.p12"
-  - name: STRIMZI_TRUSTSTORE_PASSWORD <2>
-    value: "__TRUSTSTORE-PASSWORD__"
-  - name: STRIMZI_KEYSTORE_LOCATION <3>
-    value: "/path/to/keystore.p12"
-  - name: STRIMZI_KEYSTORE_PASSWORD <4>
-    value: "__KEYSTORE-PASSWORD__"
+  - name: STRIMZI_CLUSTER_CA_CERT_SECRET_NAME <1>
+    value: "publicKeysSecret"
+  - name: STRIMZI_EO_KEY_SECRET_NAME <2>
+    value: "privateKeySecret"
```

Comment:
This is in a way API change. Can we keep the old way? If not, this needs to be mentioned in the CHANGELOG in the breaking changes section. But if you are not using the Admin Client provider from CO and UO, maybe you can keep old options as well at least as deprecated for some time?